sdk/azidentity/azure_cli_credential.go

```diff
@@ -162,32 +162,17 @@ func (c *AzureCLICredential) createAccessToken(tk []byte) (azcore.AccessToken, e
 		return azcore.AccessToken{}, err
 	}
 
-	tokenExpirationDate, err := parseExpirationDate(t.ExpiresOn)
+	// the Azure CLI's "expiresOn" is local time
+	exp, err := time.ParseInLocation("2006-01-02 15:04:05.999999", t.ExpiresOn, time.Local)
 	if err != nil {
-		return azcore.AccessToken{}, fmt.Errorf("Error parsing Token Expiration Date %q: %+v", t.ExpiresOn, err)
+		return azcore.AccessToken{}, fmt.Errorf("Error parsing token expiration time %q: %v", t.ExpiresOn, err)
 	}
 
 	converted := azcore.AccessToken{
 		Token:     t.AccessToken,
-		ExpiresOn: *tokenExpirationDate,
+		ExpiresOn: exp.UTC(),
```

Comment:
How did this bug manifest?

Reply:
I noticed it while trying to write better unit tests. It's kind of marginal as a bug in that it probably has no impact for applications beyond how expiration times appear as strings.

Reply:
I get that this unifies with the other credentials.  However, I can see why running something locally would return the expiration time in local time (vs asking Azure for a token which isn't local).  It probably doesn't matter either way, and if it improves testing then that's a win.

Reply:
I suppose that's the CLI's rationale for converting AAD's Unix timestamp to local time for display. `azidentity` is a library that generally serves internal purposes for applications though, and I believe software should use UTC internally wherever possible. I agree what we do in this case probably doesn't matter either way 😄